web-portal/test/EventCard.spec.js

```diff
@@ -86,21 +86,9 @@ describe('Card component', () => {
     expect(wrapper.html()).not.toContain('ii-location')
   })
 
-  test('renders correct event time', () => {
+  test('renders correct event time (stored in UTC, displayed in EDT)', () => {
     expect(wrapper.html()).toContain('June 1st')
-    expect(wrapper.html()).toContain('10:00am - 11:00am')
-  })
-
-  test('renders correct time when saved in UTC', () => {
-    wrapper.setProps({
-      calendar_event: Object.assign(getEvent(), {
-        date_times: [
-          { start_time: '2020-06-01T14:00:00.000Z', end_time: '2020-06-01T15:00:00.000Z' }
-        ]
-      })
-    })
-    expect(wrapper.html()).toContain('June 1st')
-    expect(wrapper.html()).toContain('10:00am - 11:00am')
+    expect(wrapper.html()).toContain('10:00am - 11:00am EDT')
```

Comment:
mysteriously, this test wasn't failing on my local machine 🤔 i didn't find this problem until this test ran on github. no idea why that is. it's an obvious one though :relaxed:

also, there used to be separate tests for datetimes stored with and without timezones. since we expect all datetimes to be stored as timezone utc with timezone specified separately, i consolidated the two tests into one updated test.